lib/kuniri/language/ruby/attribute_ruby.rb

```diff
@@ -70,54 +70,50 @@ def prepare_final_string(pString)
       end
 
       # Override
-      def handle_multiple_declaration_with_comma(pString)
-        listOfAttributes = []
-        pString = pString.split(",")
-        pString.each do |variable|
-          return nil if variable.scan(/=/).count > 1
+      def handle_line(pString)
+        if pString =~ /=/
+          pString = pString.scan(/.*=/).join("")
+          return nil if pString =~ /\./
+        end
 
-          variable = variable.scan(/.*=/).join("") if variable =~ /.*=/
+        pString = yield(pString)
+
+        return nil if pString =~ /\./
 
-          return nil if variable =~ /\./
+        pString = prepare_final_string(pString)
+        attribute = Languages::AttributeData.new(pString)
 
-          variable = prepare_final_string(variable)
-          attribute = Languages::AttributeData.new(variable)
-          listOfAttributes.push(attribute)
-        end
+        return [attribute]
+      end
 
-        return listOfAttributes
+      # Override
```

Comment:
Remove this comment. "Override" here is used in the sense of this method came from a parent class.